Libraries/LibWeb/HTML/Navigator.cpp

```diff
@@ -148,4 +149,29 @@ GC::Ref<MediaCapabilitiesAPI::MediaCapabilities> Navigator::media_capabilities()
     return *m_media_capabilities;
 }
 
+// https://w3c.github.io/battery/
+GC::Ref<WebIDL::Promise> Navigator::get_battery()
+{
+    auto& realm = this->realm();
+
+    // FIXME: This is a stub implementation. getBattery() should:
+    //        1. Query actual system battery status
+    //        2. Resolve with a BatteryManager object
+    //        3. Update when battery status changes
```

Comment:
This method is specced here:
https://w3c.github.io/battery/#the-getbattery-method

Could you please use this link instead, and copy over the spec steps? Any spec step you don't intend to implement should be marked as a `// FIXME: ...` comment.